test/unittests/parser_test.cpp

```diff
@@ -9,6 +9,14 @@ static const auto functype_void_to_void = from_hex("600000");
 static const auto functype_i32i64_to_i32 = from_hex("60027f7e017f");
 static const auto functype_i32_to_void = from_hex("60017f00");
 
+namespace
+{
+bytes push_section(uint8_t id, const bytes& section)
```

Comment:
The reason for this helper is that it makes the tests cleaner when multiple sections are pushed. I have a change in progress which enforces some checks in the parser (e.g. function and code section size must match), which require adding more sections in some tests.